tests/trading/test_cancelOrder.py

```diff
@@ -22,10 +22,12 @@
 WORSE_ORDER_ID = 6
 GAS_PRICE = 7
 
+ZEROED_ORDER_ID = '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

Comment:
This can be simplified to `bytearray(32)`.  Consider also removing this global variable and just inlining `bytearray(32)` instead of `ZEROD_ORDER_ID` (which is about as long).

Reply:
Same comment for other test files.

Reply:
I tried converting to bytearray(20) (since the orderId is currently 20 bytes, not 32), but that's throwing an error.  I think it's getting passed into the function slightly differently and may need to be formatted.

Reply:
Passing in `str(bytearray(20))` seems to work.